### PR TITLE
Bug 1245447 - fixes template search

### PR DESCRIPTION
**Pull request by fabianofranz**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1245447

### CODE REVIEW

**Comment by fabianofranz** [comment]
[merge]

File: pkg/generate/app/templatelookup.go [hunk 1]
```diff
@@ -46,7 +46,8 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 				return nil, err
 			}
 
-			for _, template := range templates.Items {
+			for i := 0; i < len(templates.Items); i++ {
```
Comment (smarterclayton) [inline pkg/generate/app/templatelookup.go]:
```
for i := range templates.Items {
  template := templates.Items[i]
}
```

**Comment by smarterclayton** [comment]
Cancel merge for review

**Comment by fabianofranz** [comment]
@smarterclayton Thanks, fixed here and in another place where I used a similar syntax.

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**